jni/CMakeLists.txt

```diff
@@ -187,6 +187,10 @@ if ("${WIN32}" STREQUAL "")
                 tests/faiss_wrapper_test.cpp
                 tests/nmslib_wrapper_test.cpp
                 tests/test_util.cpp)
+        add_executable(
+                jni_memory_test
```

Comment:
Can this be included in the general test executable?

Reply:
at 1st version, i put it into jni_test executable, but it shows that common unit test run fast, but memory_test run long time.

Reply:
that makes sense